tests/legacy-cli/e2e/tests/build/worker.ts

```diff
@@ -23,6 +23,14 @@ export default async function () {
   await expectFileToExist(workerTsConfig);
   await expectFileToMatch(snippetPath, `new Worker('./app.worker', { type: 'module' })`);
 
+  await ng('build');
+  await expectFileToExist('dist/test-project/0-es5.worker.js');
+  await expectFileToExist('dist/test-project/0-es2015.worker.js');
+
+  await ng('build', '--prod', '--output-hashing=none');
+  await expectFileToExist('dist/test-project/0-es5.worker.js');
+  await expectFileToExist('dist/test-project/0-es2015.worker.js');
+
   // console.warn has to be used because chrome only captures warnings and errors by default
```

Comment:
Should add a test that the reference to the web worker file is correct.

Example in the es2015 the es2015 worker is referenced and so on for the es5 version